R/utils.R

```diff
@@ -198,6 +198,15 @@ parse_database_error <- function(msg) {
     cnd_msg[-1],
     right = TRUE
   )
+  # Limit error mesage length to something sane
+  MAXLEN <- 5000
+  cnd_body <- sapply(cnd_body, FUN = function(x) {
+    if (nchar(x) > MAXLEN ) {
+      x <- substr(x, 0, MAXLEN)
```

Comment:
```suggestion
  # Limit error message length to something sane (#914)
  max_len <- 5000
  cnd_body <- sapply(cnd_body, FUN = function(x) {
    if (nchar(x) > max_len) {
      x <- substr(x, 0, max_len)
```

Uses snake case, clickable link (in Positron/RStudio) to GitHub archaeology in source comment, no space before end-paren.